commons-jetty/src/main/scala/com/avsystem/commons/jetty/rpc/JsonJettyRPCFramework.scala

```diff
@@ -1,12 +1,147 @@
 package com.avsystem.commons
 package jetty.rpc
 
-import com.avsystem.commons.serialization.json.{JsonStringInput, JsonStringOutput}
+import java.nio.charset.StandardCharsets
 
-object JsonJettyRPCFramework extends JettyRPCFramework {
+import com.avsystem.commons.rpc.StandardRPCFramework
+import com.avsystem.commons.serialization.GenCodec
+import com.avsystem.commons.serialization.json.{JsonReader, JsonStringInput, JsonStringOutput}
+import javax.servlet.http.{HttpServletRequest, HttpServletResponse}
+import org.eclipse.jetty.client.HttpClient
+import org.eclipse.jetty.client.api.Result
+import org.eclipse.jetty.client.util.{BufferingResponseListener, BytesContentProvider}
+import org.eclipse.jetty.http.{HttpMethod, HttpStatus}
+import org.eclipse.jetty.server.handler.AbstractHandler
+import org.eclipse.jetty.server.{Handler, Request}
+
+object JsonJettyRPCFramework extends StandardRPCFramework {
```

Comment:
I think this may be called just `JettyRPCFramework`. JSON is an implementation detail.